tensorflow/python/kernel_tests/matmul_op_test.py

```diff
@@ -70,25 +70,25 @@ def _GetTransposedMatrices(x, x_name, kwargs):
     return x
 
 class MatMulTest(test_lib.TestCase):
-  pass
+  pass # Filled in below
```

Comment:
Nit: Please put two spaces between the code (`pass`) and the comment (`#`), i.e.,
```suggestion
  pass  # Filled in below
```
Per our [Python style guide](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments).